src/server/browserContext.ts

```diff
@@ -322,7 +308,7 @@ export abstract class BrowserContext extends EventEmitter {
 
       // Bookkeeping.
       for (const listener of this._browser.options.contextListeners)
-        await listener.onContextDidDestroy(this);
+        await listener.onContextDidDestroy?.(this);
```

Comment:
Do we do this ?. already?

Reply:
As of this patch yes. See previous comments about tsconfig.json